src/node_perf_common.h

```diff
@@ -1,6 +1,7 @@
 #ifndef SRC_NODE_PERF_COMMON_H_
 #define SRC_NODE_PERF_COMMON_H_
 
+#include "aliased_buffer.h"
```

Comment:
Can you put this before the `#include "node.h"` line below?

Reply:
Sure, but any particular reason for this changes?

Reply:
@yashLadha Yes in case anyone including this header accidentally include headers they don't want (there aren't any in `aliased_buffer.h` for now, but it could be if we don't follow this pattern :) like what this PR used to do)